unlock-app/src/components/interface/VerificationStatus.tsx

```diff
@@ -18,13 +18,14 @@ import { isSignatureValidForAddress } from '~/utils/signatures'
 
 interface Props {
   config: MembershipVerificationConfig
+  setConfig: (config: MembershipVerificationConfig | null) => void
```

Comment:
Rename this to generic onScanNext function. We should expose assume anything about how the parent deals with state above. Let them pass a generic function which is called when x happens.